boxoffice/forms/item.py

```diff
@@ -8,11 +11,52 @@
 __all__ = ['ItemForm']
 
 
+def format_json(data):
+    if request.method == 'GET':
+        return json.dumps(data, indent=4, sort_keys=True)
+    return data
+
+
+def format_description(data):
+    if request.method == 'GET' and data:
+        return HTMLBeautifier.beautify(data.text, 8)
```

Comment:
What does this do?

Reply:
The ticket's description is often written in HTML to configure font-awesome icons and formatting. This just pretty prints the HTML to make the formatting easy.